.github/workflows/serverless.yml

```diff
@@ -6,13 +6,22 @@ on:
     branches: [master]
   schedule:
     - cron: 0 4 * * *
+  workflow_dispatch:
+    inputs:
+      runNightly:
+        description: 'Whether to run nightly tests'
+        required: false
+        default: 'false'
+        type: string
+
+env:
```

Comment:
Why was this moved in the file? Seems like an unnecessary change that is adding lines to review.

Reply:
What I meant is that `env` was below `concurrency` before and was moved before it, which is an unnecessary change and adds noise to the PR.